Invalid tests for Job Sequence are not formatted correctly

These:
```yaml
            sequencing_invalid:
                params:
                   ...
                    -   name: "missing input"
                        input: '"25 15 10 5"'
                        expected: "Usage: please provide a list of profits and a list of deadlines"
                    -   name: "lists different lengths"
                        input: '"1, 2, 3, 4", "1, 2, 3, 4, 5"'
                        expected: "Usage: please provide a list of profits and a list of deadlines"
```
should be like this:
```yaml
            sequencing_invalid:
                params:
                   ...
                    -   name: "missing input"
                        input: '"25, 15, 10, 5"'
                        expected: "Usage: please provide a list of profits and a list of deadlines"
                    -   name: "lists different lengths"
                        input: '"1, 2, 3, 4" "1, 2, 3, 4, 5"'
                        expected: "Usage: please provide a list of profits and a list of deadlines"
```

When I make that change, the following tests fail:
```
FAILED test/generated/test_job_sequencing.py::test_sequencing_invalid[haskell/job-sequencing.hs-lists different lengths] - AssertionError: assert '10' == 'Usage: pleas... of deadlines'
FAILED test/generated/test_job_sequencing.py::test_sequencing_invalid[python/job_sequencing.py-lists different lengths] - AssertionError: assert '10' == 'Usage: pleas... of deadlines'
```

--- a/.glotter.yml
+++ b/.glotter.yml
@@ -682,10 +682,10 @@ projects:
                         input: '""'
                         expected: "Usage: please provide a list of profits and a list of deadlines"
                     -   name: "missing input"
-                        input: '"25 15 10 5"'
+                        input: '"25, 15, 10, 5"'
                         expected: "Usage: please provide a list of profits and a list of deadlines"
                     -   name: "lists different lengths"
-                        input: '"1, 2, 3, 4", "1, 2, 3, 4, 5"'
+                        input: '"1, 2, 3, 4" "1, 2, 3, 4, 5"'
                         expected: "Usage: please provide a list of profits and a list of deadlines"
                 transformations:
                     - "strip"

--- a/archive/h/haskell/job-sequencing.hs
+++ b/archive/h/haskell/job-sequencing.hs
@@ -65,7 +65,9 @@ main = do
   let profit = headMaybe args >>= stringToListMaybe
   let deadline = tailMaybe args >>= headMaybe >>= stringToListMaybe
   let groups = zip <$> profit <*> deadline
-  let jobs = fmap (jobSequence) groups
+  let jobs = if (maybe 0 length profit) == (maybe 0 length deadline)
+             then fmap (jobSequence) groups
+             else Nothing
   case jobs of
     Nothing -> putStrLn "Usage: please provide a list of profits and a list of deadlines"
     Just n  -> putStrLn $ show $ maxProfit $ iterateJobs n

--- a/archive/p/python/job_sequencing.py
+++ b/archive/p/python/job_sequencing.py
@@ -46,6 +46,9 @@ def main(args):
     try:
         profits = input_list(args[0])
         deadlines = input_list(args[1])
+        if len(profits) != len(deadlines):
+            exit_with_error()
+
         jobs = [Job(p, d) for p, d in zip(profits, deadlines)]
         print(max_profit(jobs))
     except (IndexError, ValueError):

